Syntaxes/RSpec.tmLanguage

```diff
@@ -92,7 +94,7 @@
 		<key>behaviour</key>
 		<dict>
 			<key>begin</key>
-			<string>^\s*([fx]?(?:describe|context|feature))\b(?=.*\bdo\s*$)</string>
+			<string>^\s*([fx]?(?:RSpec\.describe|describe|context|feature))\b(?=.*\bdo\s*$)</string>
```

Comment:
That's right, this was a quick fix I had locally since we switched to the explicit call.
Fixed in dc77c35